### ISSUE TITLE
Merge 2 Input Group 

### ISSUE THREAD

**Issue by Peltier10** [arkhn/pyrog#505]
## Problem
In Toulouse usecase : two SQL Input have distinct condition because they have different id in the database. 

## Description
In Toulouse mapping, a date/time is required but the date and the time do not have the same id in the database. I need to have a FHIR dateTime attribut with 0...1 occurence. 
First solution : to have possibility to merge Input group in Pyrog

## Alternatives
Alternative solutions : in an Input group, if more than one condition are filled, possibility to choose between intersection or union. (At the moment, it is an intersection). 

![Capture d’écran 2021-04-06 à 16 37 54](https://user-images.githubusercontent.com/34452954/113729867-aee0f900-96f7-11eb-9bd3-fec243243785.png)

**Comment by simonvadee** [comment]
Thank you for reporting this, I understand the issue and agree that it needs some work to make this possible!

I personally don't think it makes sense to merge several Input groups together: since you need two columns to fill a single attribute, it fits the definition of an "input group" for which you need to merge values together (aka: using a merging script).
Therefore I would vote for implementing unions between conditions. What's your opinion @Jasopaum ?

**Comment by Jasopaum** [comment]
Agreed, the user should be able to "aggregate" conditions with ANDs and ORs 👍 
The 2 pain points being : the UI and how to model that
